### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by davidgraeff**
Signed-off-by: davidgraeff <david.graeff@web.de>

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# This file helps Github doing automatic review requests for new PRs.
+# It should always list the active maintainers of certain add-ons.
+
+# As a fallback, if no specific maintainer is listed below, assign the PR to the repo maintainers team:
+*       @openhab/2-x-add-ons-maintainers
```
Comment (kaikreuzer) [inline CODEOWNERS]: should be webui-maintainers here

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+# Add-on maintainers:
+/addons/ui/iconset/ @kaikreuzer
+/addons/ui/org.openhab.ui.basic/ @lolodomo @resetnow
+/addons/ui/org.openhab.ui.classic/ @lolodomo @resetnow
```
Comment (kaikreuzer) [inline CODEOWNERS]: @resetnow is imho not involved with the Classic UI, better put me there next to @lolodomo.

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# This file helps Github doing automatic review requests for new PRs.
+# It should always list the active maintainers of certain add-ons.
+
+# As a fallback, if no specific maintainer is listed below, assign the PR to the repo maintainers team:
+*       @openhab/2-x-add-ons-maintainers
+
+# Add-on maintainers:
+/addons/ui/iconset/ @kaikreuzer
```
Comment (kaikreuzer) [inline CODEOWNERS]: the paths are wrong, there is no `/addons` here in the repo

File: CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+/addons/ui/org.openhab.ui.basic/ @lolodomo @resetnow
+/addons/ui/org.openhab.ui.classic/ @lolodomo @resetnow
+/addons/ui/org.openhab.ui.cometvisu/ @peuter
+/addons/ui/org.openhab.ui.cometvisu.php/ @peuter
+/addons/ui/org.openhab.ui.homebuilder/ @kubawolanin
+/addons/ui/org.openhab.ui.paper/ @htreu
```
Comment (kaikreuzer) [inline CODEOWNERS]: @htreu is not actively maintaining this, so better remove this line.

Comment (kaikreuzer) [reply CODEOWNERS]: The habmin UI is missing, this is clearly for @cdjackson.
Also HABot and HABPanel are missing from @ghys.

File: CODEOWNERS (hunk 1 above)
Comment (davidgraeff) [reply CODEOWNERS]: Didn't know this is a think. Where do I look that up for the future?

Comment (wborn) [reply CODEOWNERS]: You can click the arrow to expand the [teams](https://github.com/orgs/openhab/teams) and find the most likely team. Then if you hover over the URL you can see the the ID. 

If you're a repo Admin you can also hover over the team name in Settings > Collaborators & teams.

You can validate it by typing in a comment box `@openhab/<teamname>`

Comment (ghys) [reply CODEOWNERS]: > You can click the arrow

Ha! There's an arrow indeed 😄 Good catch!

**Pull request merged**